games-fps/prboom-plus/prboom_plus-2.5.1.4.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="Enhanced clone of the classic first-person shooter Doom"
+DESCRIPTION="PrBoom+ is a Doom source port developed from the original PrBoom
+ project by Andrey Budko"
+HOMEPAGE="http://prboom-plus.sourceforge.net"
+COPYRIGHT="2016 Andrey Budko et al."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/prboom-plus/prboom-plus-$portVersion.tar.gz"
+CHECKSUM_SHA256="e0e2241d174839b107f1a42b191aa9895dc6749e477cbc850ad520fba2353b94"
+SOURCE_DIR="prboom-plus-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+
+commandBinDir="$binDir"
+if [ "$targetArchitecture" = "x86_gcc2" ]; then
+	SECONDARY_ARCHITECTURES="x86"
+	commandBinDir="$prefix/bin"
```

Comment:
What do you think of this neutral variant ?
```diff
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
-
-commandBinDir="$binDir"
+commandBinDir=$binDir
-if [ "$targetArchitecture" = "x86_gcc2" ]; then
+if [ "$targetArchitecture" = x86_gcc2 ]; then
-	SECONDARY_ARCHITECTURES="x86"
+SECONDARY_ARCHITECTURES="x86"
-	commandBinDir="$prefix/bin"
+commandBinDir=$prefix/bin
 fi
```
Yeah, the "distribution" of the quotes do not seem consistent, but that's because:
* Most of our recipes (if not all) use quotes for `ARCHITECTURES` and `SECONDARY_ARCHITECTURES`.
* Most of our recipes (if not all) do not use quotes for `commandBinDir`.

And since not having quotes in the **definition** of `commandBinDir` is still OK even if `TREE_PATH` contains special characters (such as spaces or tabs) and if `hp --no-chroot` is implemented, I think it would be nice to apply the above suggestion.